test/jdk/java/util/Formatter/Padding.java

```diff
@@ -307,7 +309,7 @@ static Arguments[] padding() {
     @ParameterizedTest
     @MethodSource
     void padding(String expected, String format, Object value) {
-        assertEquals(expected, String.format(format, value));
+        assertEquals(expected, String.format(Locale.ROOT, format, value));
```

Comment:
I suggest using `Locale.US` instead of `Locale.ROOT`. Although it works, `Locale.ROOT` is an invariant locale where theoretically you cannot assume the decimal point is a period. Using `Locale.US` assures that assumption, and also aligns with other similar test cases.

Reply:
Sure, changed to `Locale.US` in new commit.